tests/typecheck/models/test_related_fields.yml

```diff
@@ -0,0 +1,89 @@
+-   case: test_related_name_custom_manager
+    main: |
+      from app1.models import Model1
+      from app2.models import Model2
+
+      reveal_type(Model1().test)  # N: Revealed type is "app3.models.Model3_RelatedManager"
+      reveal_type(Model2().test)  # N: Revealed type is "app3.models.Model3_RelatedManager"
+      reveal_type(Model1().test2)  # N: Revealed type is "app3.models.Model4_RelatedManager"
+      reveal_type(Model2().test2)  # N: Revealed type is "app3.models.Model4_RelatedManager"
```

Comment:
Without the test these attributes will be missing, at it returns after adding the `.test` attributes